frontend/src/components/SortableTaskList.module.scss

```diff
@@ -18,7 +18,7 @@
   overflow-y: auto;
   min-height: 100px;
   height: inherit;
-  padding: 16px;
+  padding: 1px;
   padding-top: 0px;
   scrollbar-width: thin;
 
```

Comment:
```scss
.taskName {
  @extend .taskNameContainer;
  width: 175px;
}
```
This limits the task name to narrower than it would have space for. It's applied with `.leftSideDiv` from SortableTask.module.scss, where also the width should be adjusted (or maybe just extend `.taskNameContainer`, instead of this `.taskName`)

```scss
.leftSideDiv {
  @extend .taskName;
}
```